app/home/featured-work-list.tsx

```
@@ -0,0 +1,12 @@
+import React from "react"
+import client from "../../lib/sanity"
+import { homeQuery } from "../../lib/queries"
+import FeaturedWorkClient from "./featured-work-client"
+
+export const revalidate = 60 // revalidate every hour
+
+export default async function FeaturedWorkList({}) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)